Lib/test/test_argparse.py

```diff
@@ -2725,6 +2725,35 @@ def get_parser(self, required):
           -c          c help
         '''
 
+class TestMutuallyExclusiveNested(TestCase):
+    def test_format_usage(self):
+        parser = ErrorRaisingArgumentParser(prog='PROG')
```

Comment:
Is there a reason you can't use the existing test infrastructure? I think it's possible to convert these tests to follow the style of `TestMutuallyExclusiveOptionalAndPositional`: https://github.com/python/cpython/blob/82e38bd80113ecfc7aff2fb3edaf2802fe21c1fb/Lib/test/test_argparse.py#L2560

We can define a `get_parser()` method and use `usage_when_required` and `usage_when_not_required` to check output of `parser.format_help()`.